src/update.py

```diff
@@ -4,7 +4,7 @@
 import tornado.options
 import json
 import sys
-import urlparse
+from urllib.parse import urlparse
```

Comment:
This breaks for Python2.7 but you change it to 
```
try:
    from urllib.parse import urlparse
except:
    from urlparse import urlparse
```
and then it should work for both 2.7 and Python 3